SR2019/2019-02-25-mar-techday.md

```diff
@@ -0,0 +1,19 @@
+---
+to: Student Robotics 2019
+subject: March tech day reminder
+---
+
+Hi all,
+
+Just a reminder and some information on the tech day we're running on on Saturday 9th March.
```

Comment:
"on on"

Reply:
woops, looks like we send that in the last one too :man_facepalming: